components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -264,6 +264,12 @@ From there, you can select any individual rule to view its list of conditions, a
 
 ![](/images/docs/admin-projects.png)
 
+To create a project that contains all Effortless Infra nodes add a Node ingest rule with a condition that uses an attribute of Chef Server and a value of "localhost".
+
+![](/images/docs/effortless-project-rule.png)
+
+The above rule is matching on a node's Chef Server field, which is set to "localhost". This works because all Effortless Infra nodes do not have a Chef Server. Then, by adding a second condition that matches a specific Chef Policy Name, subgroups of Effortless Infra nodes can be created. 
```

Comment:
s/is matching/matches

Reply:
also I'm not sure what you mean by `subgroups`

Reply:
I have gone back and forth on that word. I was thinking that the Effortless Infra nodes are a group, then the subgroup is the nodes matching a specific policy name.

Reply:
Filtered nodes? Or maybe explicitly call them "nodes matching a specified policy name"?